scripts/flash/index.js

```diff
@@ -30,55 +31,101 @@ var NETWORKS = ["aura", "clique", "environment", "rinkeby", "ropsten"];
+    },
   },
   "designate-report": {
     method: designatedReport,
-    opts: null,
+    opts: {
+      help: {flag: true, short: "h", help: "This help, REP is given to user if needed" },
+      marketId: { required: true, short: "m", help: "Required market id" },
+      outcome: { short: "o", default: 0, help: "Outcome, needed if invalid flag isn't used, default is 0" },
+      invalid: { flag: true, short: "i", help: "Optional: amount of REP to dispute with" },
+    },
   },
   "initial-report": {
     method: initialReport,
-    opts: null,
+    opts: {
+      help: {flag: true, short: "h", help: "This help, used for Open Reporting" },
+      marketId: { required: true, short: "m", help: "Required market id" },
+      outcome: { short: "o", default: 0, help: "Outcome, needed if invalid flag isn't used, default is 0" },
+      invalid: { flag: true, short: "i", help: "Optional: amount of REP to dispute with" },
+    },
   },
   "dispute-contribute": {
     method: disputeContribute,
-    opts: null,
+    opts: {
+      help: {flag: true, short: "h", help: "This help, push time and dispute this market" },
+      marketId: { required: true, short: "m", help: "Required market id" },
+      outcome: { short: "o", default: 0, help: "Outcome, needed if invalid flag isn't used, default is 0" },
+      amount: { short: "a", help: "Optional: amount of REP to dispute with" },
+      invalid: { flag: true, short: "i", help: "Optional: amount of REP to dispute with" },
+    },
   },
   "finalize-market": {
     method: finalizeMarket,
-    opts: null,
+    opts: {
+      help: {flag: true, short: "h", help: "This help, finalize the market, it's in the correct state" },
+      marketId: { required: true, short: "m", help: "Required market id" },
+    },
   },
   "push-time": {
     method: pushTime,
-    opts: null,
+    opts: {
+      help: {flag: true, short: "h", help: "This help, push-time has been dep. use push-timestamp or set-timestamp" },
+    },
   },
   "market-info": {
     method: marketInfo,
-    opts: null,
+    opts: {
+      help: {flag: true, short: "h", help: "This help, show attributes of this market" },
+      marketId: { required: true, short: "m", help: "Required market id" },
+    },
   },
   "show-initial-reporter": {
     method: showInitialReporter,
-    opts: null,
+    opts: {
+      help: {flag: true, short: "h", help: "This help, show initial reporter address" },
+      marketId: { required: true, short: "m", help: "Required market id" },
+    },
   },
   "fork": {
     method: fork,
-    opts: null,
+    opts: {
+      help: {flag: true, short: "h", help: "This help, dispute this market all the way to fork" },
+      marketId: { required: true, short: "m", help: "Required market id" },
+    },
   },
   "approval": {
     method: approval,
-    opts: null,
+    opts: {
+      help: {flag: true, short: "h", help: "This help" },
```

Comment:
Why does `help` always start with "This help"?  I think this one and the next one should have further descriptions.